### PR TITLE
Add magnifier toggling logic

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/arctica-greeter.vala [hunk 1]
```diff
@@ -1713,4 +1730,27 @@ public class DBusServer : Object
         debug ("High-Contrast mode toggled (new state: %b), refreshing OSK, as well.", bActive);
         ToggleOnBoard (AGSettings.get_boolean (AGSettings.KEY_ONSCREEN_KEYBOARD));
     }
+
+    public void ToggleMagnifier (bool bActive, string sApplication) throws GLib.DBusError, GLib.IOError
```
Comment (sunweaver) [inline src/arctica-greeter.vala]: I don't like the idea of being able to inject any magnifier application here. In fact, _ANY_ application could be launched via this DBus interface.

Personally, we should pick one magnifier application and hard-code it here (or configure the executable in gsettings of Arctica Greeter. But this looks like the DBus client can launch $EVERYTHING.

Or do I get this code portion wrong?

**Review by sunweaver — CHANGES_REQUESTED** [review]
See inline comment...

File: src/arctica-greeter.vala (hunk 1 above)
Comment (ghost) [reply src/arctica-greeter.vala]: I originally had magnus hardcoded, but later changed the D-Bus params. Reverted.

**Review by sunweaver — APPROVED** [review]
LGTM now.

**Pull request merged**